packages/cli/src/commands/set-admin.ts

```diff
@@ -1,11 +1,35 @@
 import pickBy from 'lodash.pickby';
 import inquirer from 'inquirer';
+import { ZWeb3 } from 'zos-lib';
 
+import { parseContractReference } from '../utils/contract';
 import setAdmin from '../scripts/set-admin';
-import { fromContractFullName } from '../utils/naming';
+import {
+  promptIfNeeded,
+  networksList,
+  promptForNetwork,
+  proxiesList,
+  proxyInfo,
+  InquirerQuestions
+} from '../prompts/prompt';
 import { hasToMigrateProject } from '../prompts/migrations';
 import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesInitializer';
 
+interface PropsParams {
```

Comment:
Given that now we have some very similar interfaces shared between the `update` and the `set-admin` command, it might be a good idea to move them to a shared file `interfaces.ts` inside `src/commands` and use them the same way we do with the scripts interfaces. WDYT?

Reply:
Yes, good idea.